src/ai/susi/server/api/aaa/GetEmailSettings.java

```diff
@@ -0,0 +1,97 @@
+/**
+ *  GetApiKeys
+ *  Copyright by Shubham Gupta @fragm3
+ *
```

Comment:
This is too basic for adding additional copyright notes. Some people started in the past to add copyright notices. It does not make sense that every single file gets its own copyright now. The whole project already has a copyright.